public/data/hareom284.json

```
@@ -0,0 +1,29 @@
+{
+    "name": "Zaw Zaw Win",
+    "bio": "Web Developer",
+    "avatar": "https://github.com/hareom284.png",
+    "links": [
+        {
+            "name": "Follow me on GitHub",
+            "url": "https://github.com/hareom284",
+            "icon": "github"
+        },
+        {
+            "name": "Follow me on Twitter",
+            "url": "https://twitter.com/hareom284",
+            "icon": "twitter"
+        }
+    ],
+    "milestones": [
+        
+        {
+            "title": "Started Using twiiter",
+            "image": "https://github.com/hareom284.png",
+            "date": "June 2021",
+            "icon": "twitter",
+            "color": "red",
+            "description": "Create Content on Twitter and Share  my journey",
+            "url": "https://hareom284.me"
+        }
+    ]
+}
```

Still missing an endline.
```suggestion
}

```